yo/static_content.py

```diff
@@ -0,0 +1,61 @@
+from aiohttp import web
```

Comment:
I understand the case for serving static content during development, and I agree. What static content needs to be modified at startup, and why?